core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -441,6 +441,13 @@ oppia.directive('conversationSkin', [function() {
             $rootScope.loadingMessage = '';
             $scope.hasFullyLoaded = true;
 
+            // If the exploration is embedded, use as site language the
+            // exploration language.
+            var siteLanguage = oppiaPlayerService.getExplorationLanguageCode();
+            if ($window.GLOBALS.SUPPORTED_SITE_LANGUAGES && $scope.isIframed) {
+              $translate.use(oppiaPlayerService.getExplorationLanguageCode());
```

Comment:
English is used as default. I added the call explicitly (just in case) and some more comments.
I'm just stating what the code will do, do we want this behavior, right?